### PR TITLE
SLE-102 Help users to generate authentication token

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/server/wizard/ServerLocationWizardPage.java [hunk 1]
```diff
@@ -149,33 +165,37 @@ public void createControl(Composite parent) {
...
-      try {
-        previousPassword = ServersManager.getPassword(server);
-        if (previousPassword != null) {
-          serverPasswordText.setText(previousPassword);
+  private GridData createServerUrlField(final ScrolledForm form) {
+    Label labelUrl = new Label(form.getBody(), SWT.NULL);
+    labelUrl.setText(Messages.ServerLocationWizardPage_label_host);
+    serverUrlText = new Text(form.getBody(), SWT.BORDER | SWT.SINGLE);
+    GridData urlGd = new GridData(GridData.FILL_HORIZONTAL);
+    GridData gd = urlGd;
```
Comment (janos-ss) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/server/wizard/ServerLocationWizardPage.java]: pointless extra local variable?

Comment (henryju) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/server/wizard/ServerLocationWizardPage.java]: Good catch

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java [hunk 2]
```diff
@@ -88,6 +88,8 @@
   public static String ServerLocationWizardPage_label_password;
   public static String ServerLocationWizardPage_action_test;
   public static String ServerLocationWizardPage_action_test_tooltip;
+  public static String ServerLocationWizardPage_action_token;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java]: MINOR Rename this field "ServerLocationWizardPage_action_token" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS3008)

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java [hunk 3]
```diff
@@ -88,6 +88,8 @@
...
+  public static String ServerLocationWizardPage_action_token_tooltip;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java]: MINOR Rename this field "ServerLocationWizardPage_action_token_tooltip" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS3008)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Pull request merged**